### PR TITLE
Fix tests and forward-port service management sub-commands

### PR DESCRIPTION
**Pull request by a-palchikov**
* Fix chart tests broken with the https://github.com/gravitational/gravity/pull/1072 commit.
 * Forward port low-level system management commands for starting/stopping
gravity-specific services as well as viewing system journal.

Requires https://github.com/gravitational/trace/pull/55.

### CODE REVIEW

File: lib/app/handler/handler.go [hunk 1]
```diff
@@ -530,15 +529,6 @@ func (h *WebHandler) exportApp(w http.ResponseWriter, req *http.Request,
 	if err := json.NewDecoder(req.Body).Decode(&config); err != nil {
 		return trace.Wrap(err)
 	}
-	_, _, err = net.SplitHostPort(config.RegistryHostPort)
```
Comment (a-palchikov) [inline lib/app/handler/handler.go]: I don't believe this should do the validation.
Also, the changes to registry code seem to be allowing for optional HTTP scheme so this was wrong in enforcing the `host:port` form.

File: lib/rpc/server/agent.go [hunk 2]
```diff
@@ -91,14 +91,14 @@ func (srv *agentServer) GetRuntimeConfig(ctx context.Context, _ *types.Empty) (*
...
+	tempDir := os.TempDir()
 	config := &pb.RuntimeConfig{
 		Role:          srv.Role,
 		AdvertiseAddr: srv.Config.Listener.Addr().String(),
-		DockerDevice:  srv.DockerDevice,
 		SystemDevice:  srv.SystemDevice,
 		Mounts:        srv.Mounts,
 		StateDir:      stateDir,
-		TempDir:       srv.TempDir,
+		TempDir:       tempDir,
```
Comment (r0mant) [inline lib/rpc/server/agent.go]: Do you need a variable for it? Why not just `TempDir: os.TempDir()`?

**Review by r0mant — APPROVED** [review]

File: lib/rpc/server/agent_test.go [hunk 3]
```diff
@@ -455,3 +458,7 @@ func (r *peerStore) remove(peer Peer) {
 	defer r.Unlock()
 	delete(r.peers, peer.Addr())
 }
+
+func isPeerDeniedError(err error) bool {
+	return strings.Contains(err.Error(), "peer not authorized")
```
Comment (knisbet) [inline lib/rpc/server/agent_test.go]: Nit: "peer not authorized" should probably be a constant, or the approach where there is a pre-created error that is returned and used for comparison. var ErrPeerNotAuthorized...

File: lib/systemservice/systemd.go [hunk 4]
```diff
@@ -332,19 +345,19 @@ func (s *systemdManager) UninstallService(req UninstallServiceRequest) error {
 	serviceName := serviceName(req.Name)
 	out, err := invokeSystemctl("stop", serviceName)
 	if err != nil && !IsUnknownServiceError(err) {
-		return trace.Wrap(err, out)
+		return trace.Wrap(err)
```
Comment (knisbet) [inline lib/systemservice/systemd.go]: I'm not sure it's better to drop the out string from systemctl? Specifically on error does this not provided more context?

**Review by knisbet — APPROVED** [review]

File: lib/systemservice/systemd.go (hunk 4 above)
Comment (a-palchikov) [reply lib/systemservice/systemd.go]: Oh, I did not want that - my bad.

File: lib/rpc/server/agent_test.go (hunk 3 above)
Comment (a-palchikov) [reply lib/rpc/server/agent_test.go]: I removed the comparison with a string literal.

**Pull request merged**